bindings/nodejs/examples/client/07_get_block_data.ts

```diff
@@ -20,16 +20,16 @@ async function run() {
     });
 
     try {
-        // Create block with no payload.
-        const blockIdAndBlock = await client.buildAndPostBlock();
-        console.log('Block:', blockIdAndBlock, '\n');
+        // Fetch a block ID from the node.
+        const blockIds = await client.getTips();
+        console.log('Block IDs:', blockIds, '\n');
 
         // Get the metadata for the block.
-        const blockMetadata = await client.getBlockMetadata(blockIdAndBlock[0]);
+        const blockMetadata = await client.getBlockMetadata(blockIds[0]);
         console.log('Block metadata: ', blockMetadata, '\n');
```

Comment:
so ... we are only interested in ONE block id, right? Isn't it enough then to log only this one block id?

Reply:
Yes, might change it in another PR :+1: